src/index.js

```diff
@@ -9,20 +9,24 @@ export default function concatMap(project) {
     const innerSink = (type, data) => {
       if (type === 0) {
         innerTalkback = data
-        innerTalkback(1)
```

Comment:
why pulling has been removed from here and from the place where `1` is received from the inner source?